File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        if(x < 0)
        {
            return false;
        }
        else
        {
            String check = Integer.toString(x);
            int len = check.length();
            int z = 0;
            int y = len-1;
            while(z < y)
            {
                if(check.charAt(z) != check.charAt(y))
                {
                    return false;
                }
                z++;
                y--;
            }
            return true;
        }
    }
}

File: 9-palindrome-number/palindrome-number.go
func isPalindrome(x int) bool {
    a:= strconv.Itoa(x)
    y:=len(a)-1
    for i:=0; i<len(a)/2;i++{
        if a[i]!= a[y]{
            return false    
        }
        y--
    }
    return true
}